core/provisioning-java/pom.xml

```diff
@@ -142,6 +142,18 @@ under the License.
       <artifactId>junit-jupiter</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-core</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <!-- Mockito provides an implementation for JUnit5 extensions in the library mockito-junit-jupiter -->
+    <dependency>
```

Comment:
Please:
1. remove the comment
1. upgrade to latest version `3.2.0`: https://search.maven.org/artifact/org.mockito/mockito-junit-jupiter/3.2.0/jar
1. add the new dependency (with scope and version) in the root `pom.xml`, right after `mockito.core` and leave it here without version

Reply:
Done, should be fine now.